src/content/en/updates/2017/12/audio-worklet.md

```diff
@@ -313,27 +308,22 @@ at [Chrome WebAudio team's GitHub
 repository](https://googlechromelabs.github.io/web-audio-samples/audio-worklet/).
 
 
-## Experimental Usage and Origin Trials {: #experimental }
+## Feature Transition: Experimental to Stable {: #experimental }
 
-AudioWorklet is available on Chrome 64 (currently beta) behind the
-experimental flag. You can activate the feature with the following command
-line option:
+AudioWorklet is enabled by default for Chrome 66 or later. In Chrome 64 and 65,
+the feature is behind the experimental flag. You can activate the feature with
+the following command line option:
 
 ```
---enable-blink-features=Worklet,AudioWorklet
+--enable-blink-features=AudioWorklet
 ```
 
-Alternatively, you can go to `chrome://flags` and enable "Experimental Web
-Platform Features" and relaunch the browser. Note that this will enable the
-entire set of experimental features in the browser. Along with the
-experimental release, we have added this feature in Chrome 64 as an Origin
-Trial for all platforms. With Origin Trials, You can deploy code using
-AudioWorklet to users running Chrome 64 and get feedback from them. To
-participate in this trial please use the
-[signup form](https://docs.google.com/forms/d/e/1FAIpQLSfO0_ptFl8r8G0UFhT0xhV17eabG-erUWBDiKSRDTqEZ_9ULQ/viewform).
-
-
-
-
+Along with the experimental release, we have added this feature in Chrome 64
+and 65 as an Origin Trial for all platforms. With the Origin Trial, You can
+deploy code using AudioWorklet to users and get feedback from them. To
+participate in this trial please use the [signup
+form](https://docs.google.com/forms/d/e/1FAIpQLSfO0_ptFl8r8G0UFhT0xhV17eabG-erUWBDiKSRDTqEZ_9ULQ/viewform). This experimental program is expired on
```

Comment:
Please wrap lines at 80 chars.

Reply:
Ah. Okay. I did not know how to treat the long link so I simply added a line break after `]`.